theme.json

```diff
@@ -228,6 +262,18 @@
 					}
 				}
 			},
+			"core/group": {
+				"elements": {
+					"h2": {
+						"typography": {
+							"fontFamily": "var(--wp--preset--font-family--league-gothic)"
```

Comment:
The h2 font family is already declared here https://github.com/Automattic/sensei-theme/blob/trunk/theme.json#L562.
So I think this line is unnecessary.

Reply:
Thanks @gabrielcaires , removed it here https://github.com/Automattic/sensei-theme/pull/44/commits/139ed3b508555de4fa89647526be8dc6edadfc81